development/src/components/PeerOverlay_comp.js

```diff
@@ -0,0 +1,74 @@
+import Image from 'next/image'
+import HTML from '../../public/assets/codeEditorIcons/symbol.png'
+import CSS from '../../public/assets/codeEditorIcons/CSS3.png'
+import JS from '../../public/assets/codeEditorIcons/Group.png'
+import closeIconWhite from '../../public/assets/onboardingIcons/close_light.png';
+import Link from 'next/link';
+import React, { useState } from 'react';
+
+
+function PeerSession() {
+    const [activeLanguage, setActiveLanguage] = useState('');
+    return (
+        <div className="bg-[#2F2F3A] text-white p-6">
+            <div className=" flex flex-col">
+                <div className="flex justify-between">
+                    <div className=" font-bold text-3xl leading-8 font-nohemi">New Peer Session</div>
+                    <button><Image src={closeIconWhite} className="w-5 h-5" /></button>
+                </div>
+                <div>
+                    <div className="text-base font-nohemi font-semibold mt-8 leading-5">Session Name</div>
+                    <input
+                        type="text"
+                        name="sessionname"
+                        id="sessionname"
+                        placeholder="Enter Session Name"
+                        className="py-3 mt-2 mb-4 px-4 h-12 w-full font-nohemi font-normal text-sm bg-[#1E1E2A] text-white rounded-lg" />
+                </div>
+                <div>
+                    <div className=" font-bold mb-4 text-3xl leading-8 font-nohemi">
+                        Select Language
+                    </div>
+                    <div className="flex justify-between ">
+                        <div
+                            onClick={() => setActiveLanguage('html')}
+                            className={` w-32 h-32 flex justify-center items-center flex-col  shadow-md shadow-black rounded-md ${activeLanguage === 'html'
+                                    ? 'bg-blue-500 text-white'
+                                    : '  bg-gray-200 dark:bg-[#3D3D48]'
+                                }`}>
+                            <Image src={HTML} className="w-8 h-9 mb-4"/>
+                            <div className="text-white font-nohemi font-bold text-xl">HTML</div>
+                        </div>
+                        <div
+                            onClick={() => setActiveLanguage('css')}
+                            className={` w-32 h-32 flex justify-center items-center shadow-md shadow-black flex-col rounded-md ${activeLanguage === 'css'
+                                    ? 'bg-blue-500 text-white '
+                                    : ' bg-gray-200 dark:bg-[#3D3D48]'
+                                }`}>
+                            <Image src={CSS} alt="language-icon" className="w-8 h-8 mb-4" />
+                            <p className="text-white font-nohemi font-bold text-xl">CSS</p>
+                        </div>
+                        <div
+                            onClick={() => setActiveLanguage('js')}
```

Comment:
active language should be `javascript` not `js`

Reply:
Alright